buildSrc/src/main/java/com/uber/okbuck/core/manager/RobolectricManager.java

```diff
@@ -127,11 +131,13 @@ public void finalizeDependencies(OkBuckExtension okBuckExtension) {
       this.frameworkSdkBuildVersion = frameworkSdkBuildVersion;
     }
 
-    String getCoordinates() {
-      return "org.robolectric:android-all:"
+    String getCoordinates(String preinstrumentedVersion) {
+      return "org.robolectric:android-all-instrumented:"
```

Comment:
we can return the old format and new preinstrumentedVersion based on if `preinstrumentedVersion` is null or not. 
This will keep it backward compatible.

Reply:
We can, but this is now hardcoded in robolectric 4.7 so there's also a reason to require it and default to i3 (the default version checked in at this time). Thoughts on keeping backwards compatibility with older robo versions instead of a sane default?

Reply:
sounds good then. We can ask users to upgrade to latest robolectric in the release notes.